Example/Tests/TextTransformEquatableTests.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  TextTransformEquatableTests.swift
+//  Marker
+//
+//  Created by Harlan Kellaway on 7/7/17.
+//
+//
+
+import UIKit
+import XCTest
+@testable import Marker
+
+class TextTransformEquatableTests: XCTestCase {
+
+    func testTextStyle_isEqual_whenSameValue_andNotCustom() {
+        let none: TextTransform = .none
+        let lowercased: TextTransform = .lowercased
+        let uppercased: TextTransform = .uppercased
+        let capitalized: TextTransform = .capitalized
+        
+        XCTAssertEqual(none, none)
+        XCTAssertEqual(lowercased, lowercased)
+        XCTAssertEqual(uppercased, uppercased)
+        XCTAssertEqual(capitalized, capitalized)
+    }
+    
+    func testTextStyle_isNotEqual_whenCustom() {
```

Comment:
This is kinda weird and I don't think there is really a good way to test equality for functions. Can you leave a comment noting this in the file where you are doing `Equatable` conformation?